.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_STRAPI_URL=https://pitne-d4-app-strapi-production.up.railway.app
+REACT_APP_STRAPI_USER=buspark@bu.edu
+REACT_APP_STRAPI_PASSWORD="G6x\$WSX86H!uT&"
```

Comment:
Remove the password and add to the readme how to fill it in (Michelle can probably store it)